udf/event_code_points_to_string/udf.sql

```diff
@@ -0,0 +1,43 @@
+CREATE OR REPLACE FUNCTION udf.event_code_points_to_string(code_points ANY TYPE) AS (
+  CODE_POINTS_TO_STRING(
+    IF(
+      code_points IS NULL,
+      NULL,
+      ARRAY(
+        SELECT
+          CASE
+          WHEN
+            i IS NULL
+          THEN
+            NULL
+            -- Skip commas
```

Comment:
Commas are present between every event, so we cannot use them as index chars.

Reply:
Just wondering, do we actually have to have commas between events if they are self-contained as separate code points/characters anyway? Not having commas could simplify things, like writing regexes.

Reply:
Indeed, we do, because different events have different numbers of event properties.